it/controllers/employment/RemoveEmploymentControllerISpec.scala

```diff
@@ -221,70 +217,83 @@ class RemoveEmploymentControllerISpec extends IntegrationTest with ViewHelpers w
 
   ".submit" should {
 
-    userScenarios.foreach { user =>
+    "redirect the user to the overview page" when {
+      "it is not end of year" which {
+        lazy val result: WSResponse = {
+          dropEmploymentDB()
+          authoriseAgentOrIndividual(false)
+          urlPost(fullUrl(removeEmploymentUrl(taxYear, employmentId)), body = "", false, follow = false, headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYear)))
```

Comment:
the "false" for isWelsh can be removed since its default is false. This can be done for the other tests too